README.md

```diff
@@ -77,7 +77,8 @@ Alternatively, install [manually](/docs/Install_addon_manually.md) via Blender s
 `pip.main(["install", "pytest-blender", "--user"])`
 
 
-> **MacOS**
+> [!CAUTION]
+> **MacOS** can cause issues with running pytest normally with other projects so Linux is recommended for testing or installing Blender in an isolated environment
```

Comment:
Maybe it can be a bit more specific about what was the issue? (if we can). I think I wouldn't go as far to say that Linux is recommended since neither of us really used Linux right? 

Suggestion: 
We occasionally found some issues with the `pytest` installation for Blender, where it seemed that running pytest from a different environment would use the `pytest` installed in the Blender environment. To troubleshoot this, it may be useful to run `which pytest` from the active environment: if all is working fine, that should point to the pytest of the active environment. One can also force to run the pytest installation of the active environment by running `python -m pytest`.